docs/development/workflow/git_edit_workflow_examples.rst

```diff
@@ -52,98 +43,101 @@ Grab the latest updates to astropy
-    git push --set-upstream origin fix-1761 # tell my github acct about it
+    git checkout astropy/master -b fix-1761 # branch based on latest from GitHub
+                                            # and switch to this branch
+    git push your-user-name fix-1761        # tell my GitHub acct about it
 
-+ Make a python environment just for this fix and switch to that environment.
-  The example below shows the necessary steps in the Anaconda python
++ Make a Python environment just for this fix and switch to that environment.
+  The example below shows the necessary steps in the Miniconda/Anaconda Python
   distribution::
 
-        conda create -n apy-1761 --clone root  # Anaconda distribution only
-        source activate apy-1761
+    conda create -n apy-1761 python=3.9 # replace 3.9 with desired version
+    conda activate apy-1761
 
   If you are using a different distribution, see :ref:`virtual_envs` for
   instructions for creating and activating a new environment.
+
 + Install our branch in this environment with::
 
     pip install -e .
 
-  Remember also to use the proper version of pip or python in this context.
-
-Do you really have to set up a separate python environment for each fix? No,
-but you definitely want to have a python environment for your work on code
-contributions. Making new environments is fast, doesn't take much space and
+Do you really have to set up a separate Python environment for each fix? No,
+but you definitely want to have a Python environment for your work on code
+contributions. Making new environments is fast, does not take much space, and
 provide a way to keep your work organized.
 
+If installation fails, try to upgrade ``pip`` using ``pip install pip -U``
+command. It is also a good practice to keep your ``conda`` up-to-date by
+running ``conda update conda -n base`` when prompted to do so; maybe ``git`` too.
+
 Test first, please
 ==================
 
-It would be hard to overstate the importance of testing to Astropy. Tests are
+It would be hard to overstate the importance of testing in Astropy. Tests are
 what gives you confidence that new code does what it should and that it
-doesn't break old code.
+does not break old code.
 
 You should at least run the relevant tests before you make any changes to make
-sure that your python environment is set up properly.
+sure that your Python environment is set up properly.
 
 The first challenge is figuring out where to look for relevant tests. `Issue
 1761`_ is a problem in the `~astropy.coordinates` package, so the tests for
-it are in ``astropy/coordinates/tests``. The rest of Astropy has a similar
-layout, described at :ref:`testing-guidelines`.
+it are in ``astropy/coordinates/tests``. The rest of ``astropy`` has a similar
+layout, as described at :ref:`testing-guidelines`.
 
 Change to that directory and run the current tests with::
 
-    cd astropy/coordinates/test
+    cd astropy/coordinates/tests
     pytest
```

Comment:
This should really be:
```suggestion
    pytest astropy/coordinates/tests
```
or via tox.

Reply:
I removed instructions to keep changing directories, so they all run from top-level now.